src/ttt/ui.clj

```diff
@@ -0,0 +1,23 @@
+(ns ttt.ui
+    (:require [clojure.math.numeric-tower :as math]))
+
+(defn first-player-message
```

Comment:
I think you can change the function name to be better here, it should start with a verb since it's printing the message